### DISCUSSION TITLE
Display beatmap mapper/author in beatmap carousel cards

### DISCUSSION THREAD

**Discussion by zvyap** [ppy/osu#19310] [Ideas] [answered]
Currently, we need to select that map to see who release that map.
Why not rather show it beside the song artist's name like stable?
or just an option to enable it

Lazer:
![image](https://user-images.githubusercontent.com/52874570/180594615-79f12d3a-4226-451c-a1a1-9e1d87b43b4c.png)

Stable:
![image](https://user-images.githubusercontent.com/52874570/180594622-dce0f95b-3fbf-4350-8441-7f4128ce3bb0.png)

**Comment by peppy** [comment, marked as answer]
I can't get figma to load right now, but believe this is fixed in the new design already.

**Reply by frenzibyte** [reply]
I had checked before and it doesn't seem like it:
<img width="659" alt="CleanShot 2022-07-23 at 11 30 58@2x" src="https://user-images.githubusercontent.com/22781491/180597289-e1182772-1cd1-4dc2-9b15-b47930d2a505.png">

**Reply by Moltanicaa** [reply]
A "guest difficulty" feature has recently been added. Is it possible to add edits to the design so that it is immediately displayed whose diff it is?

**Reply by frenzibyte** [reply]
That sounds irrelevant, I think you're referring to https://github.com/ppy/osu/issues/12965.

**Reply by Moltanicaa** [reply]
No, I see that in the attached screenshot it says "mapped by daletto" on each difficulty. Why not put "difficulty from *name*" in there. It's just an idea for new design

**Reply by frenzibyte** [reply]
And I'm mentioning that thread since it's focusing on guest difficulties not being supported properly.

**Reply by Moltanicaa** [reply]
I understand, it turns out that this is already implemented, it just does not appear. I just wanted to give advice for future design (I thought it didn't work so far). Thank you.
